packages/apps/lightning-ui-docs/.storybook/webpack.config.js

```diff
@@ -23,6 +23,6 @@ module.exports = async ({ config, mode }) => {
     mode === 'PRODUCTION'
       ? path.resolve(__dirname, '../lightning-inspect/production')
       : path.resolve(__dirname, '../lightning-inspect/development');
-  config.optimization.minimize = false; // Minification seams to to break FocusManager navigation
+  //config.optimization.minimize = false; // Minification seams to to break FocusManager navigation
```

Comment:
Commenting this out to see if we no longer need this, so far it appears that we don't but would like to do some more testing before completely removing.